template-vue-ts/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
```

Comment:
Is `"module": "esnext"`required or just recommended?
I was trying to get mocha/chai working and could not run my unit tests without setting `"module": "commonjs"`  So I tried using `jest`
The result was that instead of my tests running in 2-3 seconds with mocha, and a recorded test time of 357ms, jest was taking over 14 seconds.  This was just way to slow, so I switched back to commonjs.
In my small project I have not yet encountered any issues, but I'm wondering if it will bite me later.

Reply:
`"module": "esnext"` is required because `The 'import.meta' meta-property is only allowed when the '--module' option is 'esnext' or 'system'.`

Reply:
Sorry, where is `import.meta` used? github search only shows it in this comment.

Reply:
see https://github.com/vitejs/vite#hot-module-replacement

If you need it, add 
```ts
/// <reference path="../node_modules/vite/dist/importMeta.d.ts" />
```

in your `vue-app-env.d.ts`.

And you will get typing for `import.meta.hot` and `import.meta.env`